src/material.cc

```diff
@@ -15,15 +15,17 @@ const ResourceType Material::kType = "Material";
 Material::~Material() {}
 
 Material::Ptr Material::Create(Agent* creator, double quantity,
-                               Composition::Ptr c) {
-  Material::Ptr m(new Material(creator->context(), quantity, c));
+                               Composition::Ptr c, int package_id) {
+  Material::Ptr m(new Material(creator->context(), quantity, c, package_id));
   m->tracker_.Create(creator);
   return m;
 }
 
 Material::Ptr Material::CreateUntracked(double quantity,
                                         Composition::Ptr c) {
-  Material::Ptr m(new Material(NULL, quantity, c));
+  // default package id for untracked
+  int package_id = 1;
```

Comment:
We should probably create a variable or a function with the default ID, because I thought it was 0, not 1

Reply:
Things like compositions and recipes start at 1

Reply:
OK - then we should make sure the packages start at 1 as well